src/modules/esl-tab/core/esl-tabs.ts

```diff
@@ -18,24 +20,35 @@ import {debounce} from '../../esl-utils/async/debounce';
 export class ESLTabs extends ESLBaseElement {
   public static is = 'esl-tabs';
 
+  /** List of supported scrollable types */
+  public static supportedScrollableTypes = ['disabled', 'side', 'center'];
+
   /** Scrollable mode.
-   * The following values are supported:
-   * - 'none' or not defined -  scroll behavior is disabled;
-   * - 'centered' - scroll behavior is enabled, tab is center-aligned;
+   * Supported types for different breakpoints ('disabled' by default):
+   * - 'disabled' or not defined -  scroll behavior is disabled;
+   * - 'center' - scroll behavior is enabled, tab is center-aligned;
+   * - 'side' - scroll behavior is enabled, tab is side-aligned;
    * - empty or unsupported value - scroll behavior is enabled, tab is side-aligned;
    */
-  @attr({defaultValue: 'none'}) public scrollable: string;
+  @attr({defaultValue: 'disabled'}) public scrollable: string;
 
   /** Inner element to contain {@link ESLTab} collection. Will be scrolled in a scrollable mode */
   @attr({defaultValue: '.esl-tab-container'}) public scrollableTarget: string;
 
+  private _scrollableTypeRules: ESLMediaRuleList<string>;
+
+  protected attributeChangedCallback(attrName: string, oldVal: string, newVal: string) {
+    if (!this.connected || oldVal === newVal) return;
+    if (attrName === 'scrollable') {
+      this.scrollableTypeRules = ESLMediaRuleList.parse<string>(newVal, ESLMediaRuleList.STRING_PARSER);
+      this.updateScrollableType();
+    }
+  }
+
   protected connectedCallback() {
     super.connectedCallback();
 
-    if (this.isScrollable) {
-      this.bindScrollableEvents();
-      this.updateScroll();
```

Comment:
It looks like the `this._deferredFitToViewport(this.$current, 'auto');` part from the update scroll is missed